### PR TITLE
chore: Update ROADMAP.md

### PR DESCRIPTION
**Pull request by tejal29**
Update Skaffold 2022 Roadmap.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -15,21 +15,18 @@ Feedback! We want to hear from each and every one of you: what are you personall
 
 # Product Excellence
 ### Improve Developer Experience for Helm Users
-### Identify and Address Performance Bottlenecks
+### Imrpove Skaffold Onboarding docs
```
Comment (MarlonGamez) [inline ROADMAP.md:18]:
```suggestion
### Improve Skaffold Onboarding docs
```

**Review by MarlonGamez — COMMENTED** [review]
These look good to me, I just noticed one type

**Comment by MarlonGamez** [comment]
Not sure why the CLA check is failing. Any clue @tejal29 ?

**Comment by tejal29** [comment]
> Not sure why the CLA check is failing. Any clue @tejal29 ?

yeah. Since I accepted  your suggestions, the bot fails to check when two or more authors have commits. 
We can manually resolve it.

**Pull request merged**